fe/fe-core/src/main/java/com/starrocks/memory/ProcProfileCollector.java

```diff
@@ -65,6 +67,28 @@ protected void runAfterCatalogReady() {
         deleteExpiredFiles();
     }
 
+    public String getProfileLogDir() {
+        return profileLogDir;
+    }
+
+    // AsyncProfiler depends on the native library libasyncProfiler.so, which is bundled inside the JAR.
+    // By default, it extracts this library to the /tmp directory in order to load it.
```

Comment:
zstd-jni-1.5.4-2.jar/netty-transport-native-epoll-4.1.128.Final-linux-x86_64.jar on BE side has embedded .so inside the jar, will it have the same issue?